mgr/src/certs/lightning.rs

```diff
@@ -127,8 +126,17 @@ subjectAltName = @alt_names
 DNS.1 = localhost
 IP.1 = 127.0.0.1
 IP.2 = ::1
-"#,
-    )?;
+"#
+    .to_string();
+    let mut ip_num = 3;
+    if let Some(ip) = host.ipv4_address {
```

Comment:
Mhm. If the ip address changes than the certificate is not upgraded as you can see above. I think you would need to also dump what ip addresses are currently whitelisted and regenerate the cert if it needs an update.

Reply:
Would that mean just changing the public IP in the toml and then running an 'update' or is there more to do to regenerate?

Reply:
The current code would not regenerate the certificate when the ip address changes, but if implemented correctly this should happen when "update" is run. Also an error message when one tries to connect to a host with the wrong ip in the certificate could include that an "update" command is required. However all of that can be avoided when using the ssh proxy as suggested above: https://github.com/kuutamolabs/lightning-knd/pull/444/files#r1254343285